blocks/amp-transformer.js

```diff
@@ -0,0 +1,47 @@
+
+const elementMapping = {
+	'amp-img': 'img'
+};
+const propsMapping = {
+	'class': 'className',
+	'srcset': 'srcSet'
+};
+const propsToRemove = [
+	'[value]',
+	'[class]',
+	'[disabled]'
+];
+
+/**
+ * De-AMP a React component.
+ *
+ * @param {wp.element.Component|string} element Element or string.
+ * @return {wp.element.Component|string} Transformed element or string.
+ */
+export function deamplify( element ) {
+	if ( 'string' === typeof element ) {
+		return element;
+	}
+
+	let filteredProps = {};
+
+	// Filter attributes.
+	_.each( element.props, ( v, prop ) => {
+		if ( -1 === propsToRemove.indexOf( prop ) ) {
+
+			// Eslint is giving "There should be no spaces inside this paren." error for the line.
+			if ( propsMapping[ prop ] ) { // eslint-disable-line
```

Comment:
For some reason eslint is throwing `There should be no spaces inside this paren. (space-in-parens)` error for this line. Without the spaces it's throwing spaces missing error. Not sure what I'm missing here, any thoughts?

Reply:
I've noticed that too. Seems like a bug in eslint.